libmscore/element.h

```diff
@@ -252,6 +253,8 @@ class Element : public ScoreElement {
       virtual Fraction tick() const;
       virtual Fraction rtick() const;
 
+      float beat() const;
```

Comment:
beat will be used in "is equal" construct. In these case floats a notorious unreliable because of rounding.
Is a Fraction not a better type?